storage/local/index.ts

```diff
@@ -13,18 +14,56 @@ export async function getLocalStorage() {
   if (!storage) {
     // TODO: Check browser.runtime.lastError
     const localStorage = await browser.storage.local.get(null)
-    storage = new ObservableStorage<State>(localStorage as State)
+    storage = new ObservableStorage<State>(
+      Object.assign(
+        // Empty state skeleton
+        {
+          networkActive: "",
```

Comment:
I found a bug:
`networkActive` is initially set to an empty string `""` here, so the code in [LINE 109](https://github.com/pilagod/waallet-extension/pull/207/files#diff-55f6b76efc8fecd83dc3d254bef876db761cec8df2bbdf672cf40e4e78f2b531R109) `networkActive ?? Object.keys(network)[0]` will return an `""` instead of `Object.keys(network)[0]`.

If the `PLASMO_PUBLIC_ACCOUNT_CHAIN_ID` variable is not gave in the `.env`, it will result in an `Error: No available network` error.

This issue can be resolved by using `networkActive || Object.keys(network)[0]` in [LINE 109](https://github.com/pilagod/waallet-extension/pull/207/files#diff-55f6b76efc8fecd83dc3d254bef876db761cec8df2bbdf672cf40e4e78f2b531R109):

```
    // Enable only network specified in env
    storage.set((draft) => {
      return {
        ...draft,
        networkActive: networkActive || Object.keys(network)[0],
        network,
        account,
        request: {},
        requestLog: {}
      }
    })
```

Reply:
Fixed in [855f4b0](https://github.com/pilagod/waallet-extension/pull/207/commits/855f4b0b1cdaaa9dc6e6cd9e1824e6c156ed3f2c)